### PR TITLE
Orruk Warclans - Ironjawz

### LINKED ISSUE #454: Add Orruk Warclans

### PR DESCRIPTION
**Pull request by Zirhark**
Fixes #454 

- Updated abilities
- updated artifacts
- updated command traits
- updated spells
- Added allegiances

### CODE REVIEW

File: src/army/ironjawz/abilities.ts [hunk 1]
```diff
@@ -19,6 +19,17 @@ const Abilities: TAbilities = [
     desc: `Add 1 to charge rolls for friendly IRONJAWZ units.`,
     when: [CHARGE_PHASE],
   },
+  {
+    name: `Mad as Hell`,
+    desc: `At the end of any phase, if any wounds or mortal wounds have been inflicted in that phase on an IRONJAWZ unit that is more than 9" from any enemy units, that IRONJAWZ unit can move D6".`,
+    when: [DURING_GAME],
+  },
+  {
+    name: `Ironjaqz Waaaagh!`,
```
Comment (daviseford) [inline src/army/ironjawz/abilities.ts]: Probably a typo :D

Comment (Zirhark) [reply src/army/ironjawz/abilities.ts]: lol what are you talking about? Of course he's Jaq'd...

File: src/army/ironjawz/abilities.ts [hunk 2]
```diff
@@ -19,6 +19,17 @@ const Abilities: TAbilities = [
...
     when: [CHARGE_PHASE],
   },
+  {
+    name: `Mad as Hell`,
+    desc: `At the end of any phase, if any wounds or mortal wounds have been inflicted in that phase on an IRONJAWZ unit that is more than 9" from any enemy units, that IRONJAWZ unit can move D6".`,
+    when: [DURING_GAME],
+  },
+  {
+    name: `Ironjawz Waaaagh!`,
+    desc: `You can use this command ability once per battle at the start of your combat phase if your army has a MEGABOSS' general Count up all the units that are wholly within 18" of your general and roll a die. On an 11 or lower, all friendly Ironjawz units wholly within range of the general get +1 to their attacks, if the roll is 12 or more then add +2 to their attacks instead.`,
```
Comment (exonian) [inline src/army/ironjawz/abilities.ts]: I don't have the book or anything, but this doesn't look right to me. Talks about "take a die" but rolls of 11 and 12+. Maybe it's a die per unit or something?

Comment (Zirhark) [reply src/army/ironjawz/abilities.ts]: Yeah this one I could really use the actual wording from the book, cleaned it up still so it makes a bit more sense.

Comment (wishmar) [reply src/army/ironjawz/abilities.ts]: You need to add to the roll the number on ironjawz units wholly whiting 18"" of the megaboss

**Comment by Zirhark** [comment]
I will do separate Pull requests for Bonesplitterz and what's left of the Big Waaagh. The scope of this one will be limited to the Iron Jawz.

**Comment by Zirhark** [comment]
This fixes #454

File: src/army/ironjawz/units.ts [hunk 3]
```diff
@@ -199,26 +195,16 @@ export const Units: TUnits = [
 ]
 
 // Battalions
+// TODO: Update the rest of the battalions
```
Comment (daviseford) [inline src/army/ironjawz/units.ts]: Is this TODO still relevant?

Comment (Zirhark) [reply src/army/ironjawz/units.ts]: nope, i thought i got them out all but forgot that one.

**Pull request merged**